conf/yarn-site.xml

```diff
@@ -1,6 +1,6 @@
 <configuration>
-  <property>
-    <name>yarn.nodemanager.aux-services</name>
-    <value>mapreduce_shuffle</value>
-  </property>
-</configuration>
\ No newline at end of file
+    <property>
```

Comment:
please revert. I do not see logic to this change. other files are formatted with 2 spaces. this one suddenly with 4.

Reply:
Done.